src/microbe_stage/MicrobeHUD.cs

```diff
@@ -473,8 +476,12 @@ public void ToggleHelpScreen()
 
         if (!helpScreen.Visible)
         {
-            helpScreen.Show();
+            if (!GetTree().Paused)
+                GetTree().Paused = true;
```

Comment:
Why is this if here? Why not just directly set paused to true?

Reply:
I wasn't even aware it's there, I'll remove it